indexing/src/main/java/com/google/enterprise/cloudsearch/sdk/indexing/IndexingItemBuilder.java

```diff
@@ -428,6 +429,20 @@ public IndexingItemBuilder setSearchQualityMetadataQuality(FieldOrValue<Double>
     return this;
   }
 
+  /**
```

Comment:
how was this comment added? Was this generated or added manually?

Reply:
I have added this comment, similar in style to other comments.